Symfony/CS/Fixer.php

```diff
@@ -185,15 +185,22 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
             return;
         }
 
-        if ($this->lintManager && !$this->lintManager->createProcessForFile($file->getRealpath())->isSuccessful()) {
-            if ($this->eventDispatcher) {
-                $this->eventDispatcher->dispatch(
-                    FixerFileProcessedEvent::NAME,
-                    FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_INVALID)
-                );
-            }
+        if ($this->lintManager) {
+            $lintProcess = $this->lintManager->createProcessForFile($file->getRealpath());
+            if (!$lintProcess->isSuccessful()) {
+                if ($this->eventDispatcher) {
+                    $this->eventDispatcher->dispatch(
+                        FixerFileProcessedEvent::NAME,
+                        FixerFileProcessedEvent::create()->setStatus(FixerFileProcessedEvent::STATUS_INVALID)
```

Comment:
@keradus 

I believe this would need be changed to set a different status, in order for having a different representation when displaying progress in verbose mode? Not sure.